sdk/monitor/Azure.Monitor.Ingestion/src/Generated/LogsIngestionClient.cs

```diff
@@ -55,6 +55,68 @@ public LogsIngestionClient(Uri endpoint, TokenCredential credential, LogsIngesti
             _apiVersion = options.Version;
         }
 
+        /// <summary> Ingestion API used to directly ingest data using Data Collection Rules. </summary>
+        /// <param name="ruleId"> The immutable Id of the Data Collection Rule resource. </param>
+        /// <param name="streamName"> The streamDeclaration name as defined in the Data Collection Rule. </param>
+        /// <param name="content"> The content to send as the body of the request. Details of the request body schema are in the Remarks section below. </param>
+        /// <param name="contentEncoding"> The ContentEncoding of the content. If content is gzipped, put &quot;gzip&quot;. Default value is null. </param>
```

Comment:
Can we explicitly mention that the library will not apply any compression to the passed in data for this method?

We should mention in the convenience method docs that the data will be gzipped.